package.json

```diff
@@ -54,5 +54,14 @@
     "tsconfig-paths": "^3.9.0",
     "typedoc": "^0.23.21",
     "typescript": "^4.9.3"
+  },
+  "dependencies": {
+    "@fast-check/jest": "^1.7.2",
+    "@matrixai/async-init": "^1.9.1",
+    "@matrixai/contexts": "^1.2.0",
+    "@matrixai/id": "^3.3.6",
+    "@matrixai/logger": "^3.1.0",
+    "@streamparser/json": "^0.0.17",
+    "ix": "^5.0.0"
```

Comment:
You need `@matrixai/errors` as well since you're using it in the `src`. Install `^1.2.0`.

Reply:
You also need `@matrixai/events` at `^3.0.2`.